validator/client/attest_protect_test.go

```diff
@@ -39,11 +38,23 @@ func Test_slashableAttestationCheck(t *testing.T) {
 			},
 		},
 	}
-	mockProtector := &mockSlasher.MockProtector{AllowAttestation: false}
-	validator.protector = mockProtector
+
+	m.slasherClient.EXPECT().IsSlashableAttestation(
+		gomock.Any(), // ctx
```

Comment:
In most cases I would avoid using `gomock.Any()` and pass in the actual constructed objects